core/report/markdown_templates.go

```diff
@@ -0,0 +1,61 @@
+package report
+
+const SLOTemplate = `# Reliably SLO Report
+
+Service Level Objectives identify what you should care about on your system. They are what good looks like for the users of your system. If an SLO is underperforming, it will be impacting your users in some way.
+
+<details>
+  <summary>Expand for further SLOs with Reliably</summary>
```

Comment:
Should it be 
```suggestion
  <summary>Expand for further information on SLOs with Reliably</summary>
```
as in the next one?